clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -342,6 +342,9 @@ def err_drv_sycl_missing_amdgpu_arch : Error<
 def warn_drv_sycl_offload_target_duplicate : Warning<
   "SYCL offloading target '%0' is similar to target '%1' already specified; "
   "will be ignored">, InGroup<SyclTarget>;
+def warn_drv_sycl_target_missing : Warning<
+  "linked binaries do not contain expected '%0' target; found targets: '%1'">,
+  InGroup<SyclMissingTarget>;
```

Comment:
Do we need a new warning group for this? We can just reuse `SyclTarget `here as well right?

Reply:
The diagnostic is related to usage of `-fsycl-targets` so grouping in the existing group seems reasonable.